rhba-dev-env/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,59 @@
+---
+### environment vars ###
+workspace_home:  /home/vagrant/workspace
+# https://github.com/ansible/ansible/issues/54855
+ansible_python_interpreter: /usr/bin/python3
+# same "ip" from config.yml (Vagrantfile on vagrant-ocp-bsig)
+vm_ip: 192.168.0.40
+# same "name" from config.yml (Vagrantfile)
+vm_name: workstation
+
+### kerberos config ###
+kerberos_username: mmagnani
+kerberos_password: my_kerberos_password
+
+### useful scripts ###
+build_overrides_version: '7.4.0'
```

Comment:
shouldn't be 7.5.0?

Reply:
I thought it would be better to put 7.4 because most of us are still working on this version but this is parameterizable at the beginning of the build.